packages/react/src/components/List/HierarchyList/HierarchyList.story.jsx

```diff
@@ -471,7 +471,9 @@ export const WithLargeNumberOfItems = () => (
       items={[...Array(1000)].map((_, i) => ({
         id: `item-${i}`,
         content: {
-          value: `Item ${i}`,
+          value: i === 20 ? `Item ${i} `.repeat(i + 1 * 10) : `Item ${i}`,
```

Comment:
When I was testing the story and saw the repeated value in the drag preview I was sure it was a bug. 

```suggestion
          value: i === 20 ? `Item ${i} that has an extra long label` : `Item ${i}`,
```